web/src/pages/barcoding.js

```diff
@@ -1,9 +1,335 @@
-import React from 'react'
-
-export default function BarcodingPage() {
-    return (
-        <div>
-            <h1>this is the Barcoding Page</h1>
-        </div>
-    )
+import React from 'react';
+import tw from 'twin.macro';
+import Img from 'gatsby-image';
+import { graphql } from 'gatsby';
+import styled from 'styled-components';
+
+import Layout from '../components/Layout'
+
+const HeroStyle = styled.div`
+  width: 100vw;
+  ${tw`w-screen`};
+  top: -160px;
+  margin-bottom: 200px;
+
+  .gatsby-image-wrapper {
+    top: 100px
+
+  }
+  @media screen and (max-width: 480px) {
+   .gatsby-image-wrapper {
+     width: 100vw;
+     top: 150px
+   }
+   margin-bottom: 200px;
+ }
+//  @media screen and (max-width: 700px) and (min-width: 400px) {
+//   .gatsby-image-wrapper {
+//     width: 100vw;
+//   }
+//   ${tw`w-screen`};
+// }
+`;
+
+const ImgFlex = styled.div`
+  ${tw`grid grid-cols-2 gap-8 mx-24 mb-24`};
+
+  @media screen and (max-width: 420px) {
+   .mask, .welcome {
+     width: 165px;
+   }
+   ${tw`mx-6 mb-6`}
+ }
+`;
+
+const BarGrid = styled.div`
+  ${tw`grid grid-cols-2 gap-8 mx-24 mb-24`}
+  .item-b {
+    grid-column: 2 / 3;
+    grid-row: 1 / span 2;
+  }
+  .signage {
+    grid-row: 3 / span 1;
+    grid-column: 1 / 3;
+  }
+  .pattern,
+  .lockup {
+    grid-column: 1 / 3;
+  }
+  .video {
+    grid-column: 1 / 3;
+    grid-row: 5 / span 2;
+  }
+  @media screen and (max-width: 420px) {
+    ${tw`mx-6 mb-6`}
+  }
+`;
+
+const DetailStyle = styled.div`
+  ${tw`flex flex-auto justify-between mb-24 mx-24`};
+  .portfolio-name {
+    font-weight: bold;
+    font-size: 50px;
+    line-height: 75px;
+    color: #2e444f;
+    margin-right: 50px;
+  }
+  p {
+    font-size: 22px;
+    width: 820px;
```

Comment:
If this is necessary let's add a `max-width: 100%` value as well to catch this on smaller screens